image-filters/image-filters.markdown

```diff
@@ -1064,7 +1063,7 @@ public class ColorHelper {
 ```
 
 Clearly we can’t test this with whole images. Instead we can mock the images
```

Comment:
In the interest of brevity and simplicity, I think we should just use the text you suggested above rather than introduce your new tool. @MichaelDiBernardo?